WORKSPACE

```diff
@@ -4,12 +4,40 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
 http_archive(
     name = "rules_python",
-    url = "https://github.com/bazelbuild/rules_python/releases/download/0.0.1/rules_python-0.0.1.tar.gz",
-    sha256 = "aa96a691d3a8177f3215b14b0edc9641787abaaa30363a080165d06ab65e1161",
+    sha256 = "d2865e2ce23ee217aaa408ddaa024ca472114a6f250b46159d27de05530c75e3",
```

Comment:
mypy-integration actually was incompatible with the older version of rules_python we were using. Updates to latest.

Reply:
Interesting. What was the the incompatibility? Can you link or copy-paste an error?

Reply:
```
ERROR: /private/var/tmp/_bazel_dillon/c4a378fabfbda4b24b49d8e9d5c915df/external/mypy_integration/repositories/py_repositories.bzl:20:9: //external:mypy_integration_pip_deps: no such attribute 'python_interpreter' in 'pip_import' rule
ERROR: error loading package '': in /private/var/tmp/_bazel_dillon/c4a378fabfbda4b24b49d8e9d5c915df/external/mypy_integration/repositories/pip_repositories.bzl: Encountered error while reading extension file 'requirements.bzl': no such package '@mypy_integration_pip_deps//': error loading package 'external': Could not load //external package
ERROR: error loading package '': in /private/var/tmp/_bazel_dillon/c4a378fabfbda4b24b49d8e9d5c915df/external/mypy_integration/repositories/pip_repositories.bzl: Encountered error while reading extension file 'requirements.bzl': no such package '@mypy_integration_pip_deps//': error loading package 'external': Could not load //external package
```

Reply:
Ahh. Makes sense. Cheers.